opengovsg/plumber
# PLU-89: Migrate flow substep

## Problem

Flow substep still uses MUI, so it does not reflect correctly with Figma.

## Solution

Migrate to Chakra :-)

## Things changed
- Text style change to `subhead1`
- Align icons and change the icons
- Change the success/error icon to react icons
- Change padding top and bottom
- Change the hover and active for the collapse

## Before & After Screenshots

**BEFORE**:
<img width="896" alt="image" src="https://github.com/opengovsg/plumber/assets/65110268/763021b9-2d9e-4fb8-a457-6be27f579f1c">

**AFTER**:

https://github.com/opengovsg/plumber/assets/65110268/ab6d6bc3-ef73-4854-a2cc-f9a59b77d9ff


## Tests
Regression test:
- Check that I can still continue to next step
- Check that I can still test step and publish pipe and activate workflow
